src/SFA.DAS.ASK.Web/Controllers/DeliveryPartner/PlanningMeetings/LogContactController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.StartPlanningMeeting;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.GetSupportRequest;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.SaveSupportRequest;
+using SFA.DAS.ASK.Web.Infrastructure.ModelStateTransfer;
+using SFA.DAS.ASK.Web.ViewModels.DeliveryPartner.PlanningMeetings;
+
+namespace SFA.DAS.ASK.Web.Controllers.DeliveryPartner.PlanningMeetings
+{
+    public class LogContactController : Controller
+    {
+        private IMediator _mediator { get; set; }
+
+        public LogContactController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpGet("delivery-partner/planning-meeting/log-contact/{supportId}")]
+        [ImportModelState]
+        public async Task<IActionResult> Index(Guid supportId)
+        {
+            var supportRequest = await _mediator.Send(new GetSupportRequest(supportId));
+            
+            var viewModel = new LogContactViewModel(supportRequest);
+
+            return View("~/Views/DeliveryPartner/PlanningMeetings/Logcontact.cshtml", viewModel);
+        }
+
+        [HttpPost("delivery-partner/planning-meeting/log-contact/{supportId}")]
+        [ExportModelState]
+        public async Task<IActionResult> Index(Guid supportId, LogContactViewModel viewModel)
+        {
+            var supportRequest = await _mediator.Send(new GetSupportRequest(supportId));
+            try
+            {
+                viewModel.UpdateSupportRequest(supportRequest);
+            }
+            catch
+            {
+                ModelState.AddModelError("ContactedDate", "Enter a real date");
+            }
+            if (viewModel.Email == false && viewModel.Telephone == false)
```

Comment:
Something like this for a custom data annotation might make this cleaner.  https://stackoverflow.com/a/26424791